backend/LexBoxApi/Controllers/UserController.cs

```diff
@@ -55,6 +61,7 @@ public async Task<ActionResult<LexAuthUser>> RegisterAccount(RegisterAccountInpu
             PasswordHash = PasswordHashing.HashPassword(accountInput.PasswordHash, salt, true),
             IsAdmin = false
         };
+        registerActivity?.AddTag("app.user.id", userEntity.Id);
         _lexBoxDbContext.Users.Add(userEntity);
         await _lexBoxDbContext.SaveChangesAsync();
```

Comment:
if there's an exception here where does that get recorded? do we have to catch that and record it in the activity? or will that happen automatically?

Reply:
Exceptions are recorded automatically:
![image](https://user-images.githubusercontent.com/12587509/226547156-9068297f-31ed-4a9d-b179-7f83f302bfd8.png)
![image](https://user-images.githubusercontent.com/12587509/226547129-506f7f98-4136-41f8-9a32-938d8a50a943.png)